components/helminstaller/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM golang:1.25 AS builder
+FROM --platform=$BUILDPLATFORM golang:1.25.1 AS builder
```

Comment:
I would not pin the container version... as it normally is updated automatically to the latest available patch version 

<img width="739" height="195" alt="Image" src="https://github.com/user-attachments/assets/fe3e71d8-3fac-4ce5-aa3d-4c96a86c4634" />